apps/glusterfs/operations_block_volume.go

```diff
@@ -86,8 +88,11 @@ func (bvc *BlockVolumeCreateOperation) Build() error {
 		}
 		reducedSize := ReduceRawSize(BlockHostingVolumeSize)
 		if len(volumes) > 0 {
-			bvc.bvol.Info.BlockHostingVolume = volumes[0].Info.Id
-			bvc.bvol.Info.Cluster = volumes[0].Info.Cluster
+			randSource := rand.NewSource(time.Now().UnixNano())
+			randGen := rand.New(randSource)
+			randNum := randGen.Intn(len(volumes))
```

Comment:
Var name should be something like index or randIndex. randNum (random number) is too vague.

Reply:
Done.